### ISSUE TITLE
Issue on Apple Vision Pro

### ISSUE THREAD

**Issue by justinLia** [cryptomator/ios#362]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/cryptomator/ios/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/cryptomator/ios/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

Unable to unlock in File app on Apple Vision Pro

### System Setup

- Vision OS: [1.1.2]
- Cryptomator: [2.5.2 (1228)]

### Steps to Reproduce

1. Complete the vault setup
2. The Lock Now button is dark, and it seems to be unlocked.
3. Click to enter the Files app
4. The Files app shows that it is locked, and it seems that it cannot be unlocked using Optic ID, and it cannot pop up a small passport window to enter the password like the iPad version
5. Go to Cryptomator again, and still cannot click the Lock Now button

### Expected Behavior

It should unlock in the Files app by using Optic ID or passcode, just like it does on iPad and iPhone

### Actual Behavior

The Files app shows that it is locked, and it seems that it cannot be unlocked using Optic ID, and it cannot pop up a small passport window to enter the password like the iPad version. Go to Cryptomator again, and still cannot click the Lock Now button(dark color).

### Reproducibility

Always

**Comment by tobihagemann** [comment]
Thank you for reporting this issue. Does visionOS 2 change anything? To be honest, we aren't able to test this since we don't have an Apple Vision Pro...

**Comment by justinLia** [comment]
> Thank you for reporting this issue. Does visionOS 2 change anything? To be honest, we aren't able to test this since we don't have an Apple Vision Pro...

Thank you for your reply! I just tested it again, the issue still happen with visionOS 2. Are you able to use the simulator at Xcode or something else to test it? I am happy to provide more information that you need, Thanks!

**Comment by tobihagemann** [comment]
Yes, we tested it briefly in the Simulator and can confirm that it doesn't work there either. Just wanted to make sure if it's different on a real device.

We'll look into it further and document our findings.

**Comment by iammajid** [comment]
Hi @justinLia, thank you for your offer to help. Despite our extensive research, we haven’t been able to identify the problem at this time. For now, we’ve decided to remove Vision Pro eligibility from the App Store and will not be pursuing this bug further. If circumstances change, or we gain new insights, we’ll certainly revisit this issue and provide updates.
